### ISSUE TITLE
SC 111 -- Missing alt attribute

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#2]
## URL
https://demo.fileyourstatetaxes.org/

## Issue
The image on the home page has not alt text attribute. This is required in HTML for the img element. In this instance, the image is decorative and should be coded as such. 

## Screen shot
![Image](https://github.com/user-attachments/assets/acd09941-659a-48b1-9901-e1f3a6b8a279)

## WCAG Success Criterion
[SC 1.1.1: Non-text Content (Level A)](https://w3.org/TR/WCAG21#non-text-content)

### Current
```html
<img class="fyst-home-image" src="/assets/questions/welcome....svg">

``` 

### Should be

```html
<img alt="" class="fyst-home-image" src="/assets/questions/welcome....svg">

``` 

## Also occurs on: 
- https://demo.fileyourstatetaxes.org/en/nj/landing-page

**Comment by damiansian** [comment]
Verified 

<img class="fyst-home-image" alt="" src="/assets/questions/welcome-bd39d5f35e3a006337c078879b24c3a23c0bce9242b137e905b5ff26d4712075.svg">

At the following URL: 
https://demo.fileyourstatetaxes.org/en/nj/landing-page
